cypress/support/commands.js

```diff
@@ -63,8 +66,25 @@ Cypress.Commands.add('createArticle', (title, description, body) => {
         }
       },
       headers: {
-        Authorization: `Token ${authToken}`
+        Authorization: `Token ${authToken}` 
       }
     });
   });
 });
+
+Cypress.Commands.add('deleteArticle', (slug) => {
+  cy.getCookie('auth').then((token) => {
+    if (!token) {
+      throw new Error('Auth cookie not found. Make sure you are logged in.');
+    }
+    const authToken = token.value;
+
+    cy.request({
+      method: 'DELETE',
+      url: `/api/articles/${slug}`, 
+      headers: {
+        Authorization: `Token ${authToken}` 
+      }
+    });
+  });
+});
```

Comment:
The `deleteArticle` command also lacks error handling. Consider adding a `.catch` block to handle any errors that might occur during the request.